README.md

```diff
@@ -18,9 +22,49 @@ If you'd prefer to use a local copy of the portage tree, you can point
 Licensing information can be found in the respective files, so consult
 them directly. Most ebuilds are licensed under the GPL version 2.
 
+Upstream Gentoo sources: http://sources.gentoo.org/gentoo-x86/
+
+# Removing packages
+
+Be careful when removing packages. The following section offers tips for preventing
+breakage, but they are by no means exhaustive. Gentoo's portage utilities will not
+take into account some pretty core packages (notably glibc), so take extra care
+when removing things that might interfere with those.
+
+## git log
+
+`git log <category>/<package>` will show commits that touched that directory. These can give
+clues about why a package was included in the first place and where to look to ensure it really
+is unused.
+
+## equery d
+
+`equery d -a <package-name>` will tell you what packages depend on `package-name`. It will
+also generate a lot of false positives, since it considers all dependencies for all use flags,
+even ones we do not use, such as `test`.
+
+## emerge --emptytree
+
+`emerge --pretend --verbose --emptytree <package-name>` _should_ give a list of all the dependencies
+for a given package. After removing the ebuild and package, make sure its removal doesn't break
+the packages `board-packages` and `sdk-depends`. Remember to use the `emerge-<arch>-usr` commands
+to check `board-packages` and `emerge` to check `sdk-depends`.
+
+The ebuild must be removed from `portage-stable` and the package must be removed locally or else
+the package will be silently elided in the dependency list. To see if there are any packages
```

Comment:
This appears to say that if I leave the ebuild in place and leave the package installed, it won't show up in the dep list, which doesn't sound right. Is the intent to say that the problem occurs if I remove the ebuild but leave the package installed?